ONSdigital/rm-reporting
# added underscores to column name references

# Motivation and Context
The column names in case and collection exercies services have recently had underscores added to them, but the reporting service didn't have these changes and was causing issues in preprod. This PR updates the reporting to have the correct column names.

# What has changed
Added underscores to case and collection exercise column names.

# How to test
Make sure case service is at least 11.1.42.

Click on a survey and click the response chasing report link. You should download a file without getting a 500 error.

# Links
[Card](https://trello.com/c/0e2SsziS)
